docs/release-notes.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Release notes
+---
+
+## Version 0.1
+
+- First version of repo.
+- BigNum features:
+  - Two constructors (long long, string).
+  - Operator overload.
+  - int and long long conversion.
+  - Basic operations:
+    - Division.
+    - Multiplication.
+    - Modulus.
+    - Addition.
+    - Subtraction.
```

Comment:
This file is not needed, as repo hasn't a release yet.

Reply:
Done